csharp/NEP17/NEP17.cs

```diff
@@ -10,28 +11,49 @@ namespace Neo.SmartContract.Examples
     [ManifestExtra("Description", "This is a NEP17 example")]
     [SupportedStandards("NEP-17")]
     [ContractPermission("*", "onNEP17Payment")]
-    public partial class NEP17Demo : Framework.SmartContract
+    public partial class NEP17Demo : Nep17Token
     {
-        #region Token Settings
-        static readonly ulong MaxSupply = 10_000_000_000_000_000;
-        static readonly ulong InitialSupply = 2_000_000_000_000_000;
-        static readonly UInt160 Owner = "NiNmXL8FjEUEs1nfX9uHFBNaenxDHJtmuB".ToScriptHash();
-        static readonly ulong TokensPerNEO = 1_000_000_000;
-        static readonly ulong TokensPerGAS = 1;
-        #endregion
+        [InitialValue("NhGobEnuWX5rVdpnuZZAZExPoRs5J6D2Sb", ContractParameterType.Hash160)]
+        private static readonly UInt160 owner = default;
+        private static readonly byte[] ownerKey = "owner".ToByteArray();
```

Comment:
Why not use `byte`?

Reply:
> Why not use `byte`?

What's wrong with byte? You'd like ByteString?

Reply:
I think we should use `byte`.

Reply:
It is not solved.

Reply:
> It is not solved.

Done.